### PR TITLE
Check Whether child  pse attendance is >=21

### PR DESCRIPTION
**Pull request by Rohit25negi**
Add A new column to check how many children have pse days attended >=21. This will be used in Service delivery dashboard.
This does not have any prerequisite. Once approved can be deployed.  and let  aggregated.

@calellowitz @emord

### CODE REVIEW

**Review by calellowitz — CHANGES_REQUESTED** [review]
This migration will conflict but otherwise looks good.

**Comment by Rohit25negi** [comment]
@calellowitz  Resolved Conflict. ready for next review cycle/approval :P.

File: custom/icds_reports/migrations/0092_pse_days_21.py
Comment (emord) [inline custom/icds_reports/migrations/0092_pse_days_21.py]: unfortunately I think this will also conflict with migration in #23109

Its a bit difficult to solve this well in django. My general strategy previously has been to structure my PRs such that all the migrations are in one PR, or the PRs build off of one another when there is a some order that they will be merged in. I know that most of the PRs you've submitted can be merged in parallel, so I'm not sure on the best way to structure them so that you don't run into this issue

Comment (Rohit25negi) [reply custom/icds_reports/migrations/0092_pse_days_21.py]: yeah, So I am following a simple not so smart strategy  to do this. Before merging any PR with migration I first merge it master then again run `migrate` in local and see if any conflict is there. if there is a conflict, resolve that and then push 1 commit after resolving that. By this I  ensure that its not conflicting. 
So for this. This is going to conflict if https://github.com/dimagi/commcare-hq/pull/23109 is merged first. Whatever branch will be merged later, I will be performing the above steps to remove the conflict. @emord

Comment (Rohit25negi) [reply custom/icds_reports/migrations/0092_pse_days_21.py]: I think, I Like your Idea of making all the migrations in one PR then do the work later in later PRS, which I can use for  working in any realease work or any coherent work

**Review by calellowitz — APPROVED** [review]

**Comment by Rohit25negi** [comment]
Do not merge this now!

**Comment by calellowitz** [comment]
@Rohit25negi  I added the "do no merge" label but that is a good way to let people know not to merge a pr in the future

**Pull request merged**
